src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -25,6 +25,7 @@
 #ifndef SHARE_UTILITIES_GLOBALDEFINITIONS_HPP
 #define SHARE_UTILITIES_GLOBALDEFINITIONS_HPP
 
+#include "utilities/bitCast.hpp"
```

Comment:
I think including bitCast here is problematic.  globalDefinitions is included nearly everywhere, while
bitcasting is only needed in a few.  There is also a high risk of circular include dependencies when
adding anything to this file's include list.  It looks like this addition happens to work today, but...
And as noted elsewhere, I'm not sure some of the changes being made here to use bit_cast should
be made anyway.  And if they should, perhaps the relevant code ought to be split out of this file.

Reply:
Reverted changes to globalDefinitions.hpp.